smartapps/ady624/webcore.src/webcore.groovy

```diff
@@ -559,68 +308,82 @@ private pageFinishInstall() {
 	dynamicPage(name: "pageFinishInstall", title: "", install: true) {
 		section() {
 			paragraph "Excellent! You are now ready to use ${handle()}"
-        }
-        section("Note") {
-            paragraph "After you tap Done, go to the Automation tab, select the SmartApps section, and open the '${app.label}' SmartApp to access the dashboard.", required: true
-            paragraph "You can also access the dashboard on any another device by entering ${domain()} in the address bar of your browser.", required: true
-        }
-        section() {
-            paragraph "Now tap Done and enjoy ${handle()}!"
+		}
+		section("Note") {
+			paragraph "After you tap Done, go to the Automation tab, select the SmartApps section, and open the '${app.label}' SmartApp to access the dashboard.", required: true
+			paragraph "You can also access the dashboard on any another device by entering ${domain()} in the address bar of your browser.", required: true
+		}
+		section() {
+			paragraph "Now tap Done and enjoy ${handle()}!"
 		}
 	}
 }
 
 def pageSettings() {
-    //clear devices cache
+	//clear devices cache
 	dynamicPage(name: "pageSettings", title: "", install: false, uninstall: false) {
 		section("General") {
 			label name: "name", title: "Name", state: (name ? "complete" : null), defaultValue: app.name, required: false
 		}
 
-        def storageApp = getStorageApp()
-        if (storageApp) {
-			section("Available devices") {
-	        	app([title: isHubitat() ? 'Do not click' : 'Available Devices', multiple: false, install: true, uninstall: false], 'storage', 'ady624', "${handle()} Storage")
-	        }
+		def storageApp = getStorageApp()
+		if (storageApp) {
+			section("Storage Application") {
+				app([title: isHubitat() ? 'Do not click - App Launchs automatically' : 'Available Devices', multiple: false, install: true, uninstall: false], 'storage', 'ady624', "${handle()} Storage")
+			}
 		} else {
 			section("Available devices") {
 				href "pageSelectDevices", title: "Available devices", description: "Tap here to select which devices are available to pistons"
 			}
-        	    }
-        
-        section("Fuel Streams"){
-            input "localFuelStreams", "bool", title: "Use local fuel streams?", defaultValue: isHubitat() ? true : false, submitOnChange: true
-            if(settings.localFuelStreams){
-                href "pageFuelStreams", title: "Fuel Streams", description: "Tap here to manage fuel streams"                
-            }         	
-        }
-        
+		}
+
+		section(sectionTitleStr("Fuel Streams")){
+			input "localFuelStreams", "bool", title: "Use local fuel streams?", defaultValue: (settings.localFueStreams != null) ? settings.localFueStreams : isHubitat() ? true : false, submitOnChange: true
```

Comment:
I don't think the default value should depend on the current setting from itself

Reply:
If you don't, it always turns on on each enter of the page.  It is not good form to change values just for entering a page.  IF you turn off, leave it off.

Reply:
I'm not seeing this behavior in webcore or this test app:

```
definition(name: "Test UI defaults", namespace: "jp0550", author: "jp0550", iconUrl: "about:blank", iconX2Url: "about:blank", description: "ui app")

preferences{
	section {
		input "boolean", "bool", defaultValue: true
	}
}
```